dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties

```diff
@@ -15,19 +15,33 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=The Liberty Audit feature is used to report and track auditable events to ensure the integrity of your system. \
-The Liberty Audit feature introduces an infrastructure which serves two purposes: \
- - Confirming the effectiveness and integrity of the existing configuration \
- - Identifying areas where improvement to the configuration may be needed  \
-The Liberty Audit feature has the ability to capture the following auditable events: \
- - Basic authentication \
- - Start and stop of the Audit service \
- - Form login \
- - Client certificate login \
- - Servlet runAs delegation \
- - Form logout \
- - Servlet 3.0 login, logout and authenticate APIs \
-The Liberty Audit feature supports the Cloud Auditing Data Federation (CADF) event model.  The CADF model describes a data model and associated schema definitions for an audit event. \
-The feature provides a default implementation, the AuditFileHandler, which emits human-readable audit records to a file-based log.   Each audit record is emitted in JSON format. \
-
-
+description=The Liberty Audit feature is used to report and track auditable events to ensure the integrity of your system. \n\
+The Liberty Audit feature introduces an infrastructure which serves two purposes: \n\n\
+ - Confirming the effectiveness and integrity of the existing configuration \n\
+ - Identifying areas where improvement to the configuration may be needed  \n\n\
```

Comment:
Change
"- Identifying areas where improvement to the configuration **may** be needed"
to
"- Identifying areas where improvement to the configuration **might** be needed"